src/app/butter-provider/yts.js

```diff
@@ -120,14 +131,19 @@ class YTSApi extends Generic {
     if (filters.sorter && filters.sorter !== 'last added') {
       switch (filters.sorter) {
       case 'popularity':
-      case 'trending':
         params.sort_by = 'download_count';
         break;
+      case 'trending':
+        params.sort_by = 'peers';
+        break;
       default:
         params.sort_by = filters.sorter;
       }
     }
 
+    const rating = filters.rating === 'All' ? '0' : filters.rating;
+    params.minimum_rating = rating || '0';
+
```

Comment:
Any reason I'm missing why this isnt
```
if (filters.rating && filters.rating !== 'All') {
  params.minimum_rating = filters.rating;
}
```

Do we need to send a filter parameter when the user hasnt this to 'All'?

Reply:
I'll check that